timemachine/cpp/src/kernels/k_nonbonded.cuh

```diff
@@ -701,76 +725,85 @@ void __global__ k_nonbonded_unified(
     unsigned long long * __restrict__ du_dx,
     unsigned long long * __restrict__ du_dp,
     unsigned long long * __restrict__ du_dl_buffer,
-    unsigned long long * __restrict__ u_buffer) {
-
+    unsigned long long * __restrict__ u_buffer
+) {
+    const int stride = gridDim.x;
```

Comment:
Does replacing gridDim.x with NONBONDED_KERNEL_BLOCKS make this any faster? (compile time constant vs runtime constant).

Reply:
Makes no apparent difference

Reply:
Thanks for checking!